src/app/services/ccd-config/ccd-case.config.ts

```diff
@@ -160,6 +160,10 @@ export class AppConfig extends AbstractAppConfig {
     return this.workallocationUrl;
   }
 
+  public getRefundsUrl(): string {
```

Comment:
What is the purpose of this? Is this to be changed at some point?

Reply:
The new CCD config has a required field getRefundsUrl.